### ISSUE TITLE
Requirements and test

### ISSUE THREAD

**Issue by dbaio** [viraptor/phply#43]
Hello.

It seems that after bd2491b4d3f9fc64886b00e577139bd897398373 , `ply` needs to be on `setup_requires` as well. 

Is there any reason to remove the test suite from sdist? On FreeBSD we prefer to fetch Python packages through PyPI and now the test is not present there.

It would be great if you can change it back.

And thanks for your work.

Regards.

**Comment by viraptor** [comment]
I wouldn't say setup requires ply. It's completely optional unless you intend to package system-wide.

Is there some situation where you find it should be required?

**Comment by dbaio** [comment]
Thanks for reverting the tests.

I think after this https://github.com/viraptor/phply/commit/bd2491b4d3f9fc64886b00e577139bd897398373 ply is necessary for building.

```
running build
running gen_parsetab
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "setup.py", line 66, in <module>
    'gen_parsetab': GenerateParsetab,
  File "/usr/local/lib/python2.7/site-packages/setuptools/__init__.py", line 129, in setup
    return distutils.core.setup(**attrs)
  File "/usr/local/lib/python2.7/distutils/core.py", line 151, in setup
    dist.run_commands()
  File "/usr/local/lib/python2.7/distutils/dist.py", line 953, in run_commands
    self.run_command(cmd)
  File "/usr/local/lib/python2.7/distutils/dist.py", line 972, in run_command
    cmd_obj.run()
  File "/usr/local/lib/python2.7/distutils/command/build.py", line 127, in run
    self.run_command(cmd_name)
  File "/usr/local/lib/python2.7/distutils/cmd.py", line 326, in run_command
    self.distribution.run_command(command)
  File "/usr/local/lib/python2.7/distutils/dist.py", line 972, in run_command
    cmd_obj.run()
  File "setup.py", line 19, in run
    from phply.phpparse import make_parser
  File "phply/phpparse.py", line 9, in <module>
    from . import phplex
  File "phply/phplex.py", line 7, in <module>
    import ply.lex as lex
ImportError: No module named ply.lex
*** Error code 1
```

**Comment by viraptor** [comment]
Fixed in 1.2.4

**Comment by dbaio** [comment]
Thank you so much
